src/scripts/charts/AxisChart.js

```diff
@@ -349,6 +349,10 @@ export default class AxisChart extends BaseChart {
 					};
 				});
 
+				const chartPosition = this.chart_wrapper.getBoundingClientRect();
+				x += chartPosition.left;
+				y += chartPosition.top - 15;
```

Comment:
What does the `15` signify? Perhaps we can figure out it's relation to the translate measurements, so it doesn't have to be hardcoded :)

Reply:
Fifteen pixels here creates a margin between pointer arrow and its target. That margin doesn't look good on heatmap, so it's here to affect only other chart types.